templates/types/streaming/nextjs/app/components/ui/chat/chat-input.tsx

```diff
@@ -63,17 +65,12 @@ export default function ChatInput(
     attachments: JSONValue[] | undefined,
   ) => {
     e.preventDefault();
-    props.append!(
-      {
-        content: props.input,
-        role: "user",
-        createdAt: new Date(),
-        annotations: attachments,
-      },
-      {
-        data: { imageUrl, csvFiles: files },
-      },
-    );
+    props.append!({
+      content: props.input,
+      role: "user",
+      createdAt: new Date(),
+      annotations: attachments,
```

Comment:
@thucpn these are just the annotations of the current message right?

but the request also sends all other annotations, where is this handled?

Reply:
e.g. can we stop re-sending the annotations that are not from user messages (like tools, etc.)

Reply:
ah then we better leave this as an optimization step, we want the code to be as small as possible

Reply:
@thucpn good job overriding the submit!